config/ompi_configure_options.m4

```diff
@@ -255,5 +255,16 @@ AC_DEFINE_UNQUOTED([OMPI_BUILD_FORTRAN_F08_SUBARRAYS],
                    [$OMPI_BUILD_FORTRAN_F08_SUBARRAYS],
                    [Whether we built the 'use mpi_f08' prototype subarray-based implementation or not (i.e., whether to build the use-mpi-f08-desc prototype or the regular use-mpi-f08 implementation)])
 
+AC_ARG_ENABLE([mpi-io],
+    [AC_HELP_STRING([--disable-mpi-io],
+        [Disable built-in support for MPI-2 I/O, likely because
+         an externally-provided MPI I/O package will be used.
+         Default is to use the internal component system and
+         its specially modified version of ROMIO])])
```

Comment:
The default is now to use OMPIO, not ROMIO, right?

I know you just copied this help text, but I think it is now stale / outdated and needed to be updated anyway... 😄